source/documentation/introduction_to_the_vm_portal/topics/Graphical_User_Interface_elements.adoc

```diff
@@ -8,23 +8,50 @@ image::VM_screen.png[VM Portal screen]
-*** *About*: VM Portal release information
-*** *Log out*: To log out of the VM Portal
+The header bar contains the following buttons:
 
-** The btn:[Messages] drop-down button displays messages from the system.
-
-* image:2.png[title="Toolbar"] `Toolbar`
+** btn:[Refresh] - manually refreshes the display. See also Refresh Interval.
+** btn:[Account Settings] -  enables setting of logged-in user preferences that are persisted on the server.
+*** btn:[Reset settings] - allows you to reset all options back to initial installation defaults.
+*** General - displays the user name, email address, and allows you to change the UI language.
++
+[NOTE]
+====
+Changing the language setting will override the language chosen in a landing page or by a URL query. See also Advanced Options.
+====
+
+*** Refresh Interval - set the automatic UI refresh rate interval.
+*** Notifications - allows you to mute all notifications for a given time interval. This option is not server persisted, so a page reload will clear this settings.
+*** Console - settings are applied globally for all VMs.
+.. select your preferred console. This will be the first console option  displayed in the VM card (VNC console,  VNC console (Browser),  SPICE console, Remote desktop).
+.. Connect automatically - automatic console login option. Enables automatically connecting to the VM's console of the chosen VM after the user logs in to the VM Portal.
+.. VNC Options
+.. VNC (Browser) Options
+.. SPICE Options
+.. Serial Console Options - SSH Key for authentication.
+*** Advanced Options - enable language settings to persist on the server. Enable the the chosen language/URL query language parameter on the landing page to override the chosen user settings language after each login. This enables compatibility with previous language settings behavior.
+** btn:[User] - displays the user name of the currently logged in user.
+*** About - VM Portal release information.
+*** Log out - to log out of the VM Portal.
+** btn:[Notifications] - displays notifications.
+
+* `Toolbar`
 +
-The toolbar contains buttons that allow you to perform additional actions.
+The toolbar bar allows you to create new virtual machines (btn:[Create Virtual Machine] button is only available to authorized users), and search for virtual machines using a filter for the Name, Status, or Operating System of virtual machines or pool entities. The number of currently displayed virtual machines or pool entities and the total number available is displayed.
 
-* image:3.png[title="Virtual machines pane"] `Virtual machines pane`
+* `Virtual machines pane`
 +
-The virtual machines pane displays the icon, operating system, name, state, and management icons of each virtual machine and pooled virtual machine.
+The virtual machines pane displays the icon, operating system, name, status, and management options for each virtual machine or pooled virtual machine.
+
+** VM management options:
```

Comment:
the default action (the one visible without opening the drop down) depends on the VM/pool state/type i.e. for shutdown VM it will be `Run` but for running it will be the most preferred console (if any).